magento/pwa-studio
# [PB] [Docs] PB-38: Initial setup in PWA docs

## Description

This PR sets up the Page Builder component documentation within PWA's documentation framework. It adds markdown host files for importing and displaying the Page Builder source code docs as well as configurations for the navigation/TOC contents.

## Related Issue

https://jira.corp.magento.com/browse/PB-38

## Acceptance

### Verification Stakeholders

@davemacaulay 
@jcalcaben 

### Verification Steps

1. Copy pagebuilder.yml and top-nav.yml into _data (overwriting existing top-nav.yml).
2. Copy entire page-builder directory into src.
3. Cd to pwa-docs and run: npm run develop.
4. See PWA docs as shown in screen shot.

## Screenshots / Screen Captures (if appropriate)

![image](https://user-images.githubusercontent.com/1828494/65685174-055f7880-e027-11e9-97f1-5ca4357aa07c.png)

![image](https://user-images.githubusercontent.com/1828494/65685206-16a88500-e027-11e9-8325-b13944a536d2.png)
